crates/apub/src/objects/person.rs

```diff
@@ -136,34 +156,35 @@ impl ApubObject for ApubPerson {
     context: &LemmyContext,
     request_counter: &mut i32,
   ) -> Result<ApubPerson, LemmyError> {
-    let person_form = PersonForm {
+    // TODO Maybe a better way to do this? Same for community and site from_apub
+    let domain = generate_domain_url(person.id.inner())?;
+    let instance = blocking(context.pool(), move |conn| Instance::create(conn, &domain)).await??;
```

Comment:
Could also be cleaner if both of these things were handled in a single helper method.